ci/vcpkg/ports.patch

```diff
@@ -1,145 +1,3 @@
-diff --git a/ports/aws-c-auth/vcpkg.json b/ports/aws-c-auth/vcpkg.json
-index dc8f75e8..be703324 100644
---- a/ports/aws-c-auth/vcpkg.json
-+++ b/ports/aws-c-auth/vcpkg.json
-@@ -4,7 +4,7 @@
-   "port-version": 1,
-   "description": "C99 library implementation of AWS client-side authentication: standard credentials providers and signing.",
-   "homepage": "https://github.com/awslabs/aws-c-auth",
--  "supports": "!arm & !uwp",
```

Comment:
They changed `!arm` to `!(windows & arm)` e.g. https://github.com/microsoft/vcpkg/blob/master/ports/aws-c-auth/vcpkg.json#L7
So I think these are no longer necessary?

Reply:
Then it indeed shouldn't be necessary.